73.md

```diff
@@ -0,0 +1,190 @@
+NIP-73
+======
+
+Spreadsheets
+------------
+
+`draft` `optional`
+
+This NIP provides a simple way to save spreedsheets on Nostr. 
+
+Event kind `35337` describes a workbook with `data` tags that contain the value of each cell as well as optional styling. 
+
+```js
+{
+  "kind": 35337,
+  "tags": [
+    ["d", "<unique identifier>"],
+    ["title", "Name of this spreadsheet"], // public title
+    ["data", "<sheet name>", "<column letter>", "<row number>", "<value>"],
+    ["data", "<sheet name>", "<column letter>", "<row number>", "<value>", "<style>"],
```

Comment:
The issue is how to not break the map between these names and the formulas, since formulas use the names instead of indexes... If one client wants to start with A while others want to start with a letter of their alphabet, they wouldn't be able to translate indexes without mapping back to A first and then translating the formula.